composer.json

```diff
@@ -21,11 +21,7 @@
     ],
     "require": {
         "php": ">=5.3.3",
```

Comment:
I'm fine with that, will match it with the Symfony 3 PHP requirement, which is `>=5.5.9`